src/components/Steps/__stories__/steps.stories.mdx

```diff
@@ -136,23 +144,62 @@ Steps with a number view or gallery view.
 
 ## Use cases and examples
 
+### Navigable Steps
+
+Navigable steps example.
+
+<Canvas>
```

Comment:
Please check this story with Meirav (design review) and see if she prefer to change something.
Maybe we want also to fix the overview story as well (the problem with the overview story is that the code example of it will not show how we create all the variables before we return the component because of storybook mechanism)